OrangebeardLogger.cs

```diff
@@ -227,17 +183,58 @@ private void PopulateAttachmentData(ref string message, ref string attachmentMim
                             attachmentData = File.ReadAllBytes(filePath);
                             attachmentFileName = Path.GetFileName(filePath);
```

Comment:
Somehow this no longer works? The whole path is expected in attachmentFileName for the logToOrangebeard function to find the file. Not sure if this is new or never worked in the first place (however I see attachments from filename filters in earlier runs)

Reply:
Should be fixed now; passing the full path instead of just the file name.